demo/index.html

```diff
@@ -89,6 +89,10 @@
         <div>B</div>
         <paper-slider class="blue" value="211" max="255" editable></paper-slider>
       </div>
+      <div class="center horizontal layout">
+      <div>O</div>
+      <paper-slider max="1.0" step="0.01" editable></paper-slider>
+      </div>
```

Comment:
@arthurevans the `O` meant `Opacity` but I think `&alpha;` is more suitable for the `alpha` channel of a `RGBA` color.